interface/ceed-qfunction.c

```diff
@@ -163,6 +164,35 @@ int CeedQFunctionCreateInteriorByName(Ceed ceed,  const char *name,
   return 0;
 }
 
+/**
+  @brief Create an identity CeedQFunction. Inputs are written into outputs in
+           the order given.
```

Comment:
Could the description explain why one might use this? Perhaps sketch an example where this enables a backend to fast-path?

Reply:
Updated